src/main/java/task/Tasklist.java

```diff
@@ -0,0 +1,93 @@
+package task;
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Tasklist {
+    ArrayList<Task> taskList;
+
+    public Tasklist() {
+        this.taskList = new ArrayList<>();
+    }
+
+    
+    /** 
+     * Returns a string of all the tasks
+     * @return String
+     */
+    public String getList() {
+        // init string to return
+        String retString = "";
+
+        for (int i = 0; i < taskList.size(); i++) {
+            retString += (i + 1) + ". " + taskList.get(i) + "\n";
+        }
+        return retString;
+    }
+
+    /** 
+     * Returns a string of all the tasks matching regex
+     * @return String
+     */
+    public String getMatchRegex(String s) {
+        // init string to return
+        String retString = "";
+
+        Pattern pattern = Pattern.compile(s);
+        int counter = 1;
+
+        for (int i = 0; i < taskList.size(); i++) {
+            Matcher matcher = pattern.matcher(taskList.get(i).toString());
+            if (matcher.find()) {
+                retString += counter + ". " + taskList.get(i) + "\n";
+                counter++;
+            }
+        }
+        return retString;
+    }
+
+    
+    /** 
+     * Return string of the task at that position
+     * @param pos
+     * @return Task
+     */
+    public Task getStr(int pos) {
+        return taskList.get(pos);
+    }
```

Comment:
I think this could be renamed as `getTask()` instead since it's returning a `Task` and not a `String`.